lib/pharos/resources/hcloud/hcloud-csi.yml.erb

```diff
@@ -135,7 +135,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: hcloud-csi-driver
-          image: kontenapharos/hcloud-csi-driver:1.1.4-kontena.1
+          image: hetznercloud/hcloud-csi-driver:1.1.5
```

Comment:
Should we "prefix" this with `docker.io/...`?

Reply:
Does not hurt, fixed.